src/main/datatypes/MQ2AdvLootType.cpp

```diff
@@ -345,7 +347,49 @@ bool MQ2AdvLootItemType::GetMember(MQVarPtr VarPtr, const char* Member, char* In
 		Dest.Type = pBoolType;
 		return true;
 
-	default: break;
+	case AdvLootItemMembers::Status:
+	{
+		Dest.Type = pStringType;
+
+		std::string tmp_str = "INVALID";
```

Comment:
Just copy the literal in the switch, there's no reason to allocate a temporary string